android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -231,6 +232,21 @@ public void sendNotification(Bundle bundle) {
                 notification.setVibrate(new long[]{0, vibration});
             }
 
+            int badge = -1;
+            if (bundle.containsKey("badge")) {
```

Comment:
Actually, after a read of the RN [docs](https://facebook.github.io/react-native/docs/pushnotificationios.html) I think it's applicationIconBadgeNumber when included in a notification. 

And there's also a static method `setApplicationIconBadgeNumber`.